base/linalg/dense.jl

```diff
@@ -509,7 +509,7 @@ function logm(A::StridedMatrix)
         end
     end
 
-    if isreal(A) && ~np_real_eigs
+    if all(isreal, A) && ~np_real_eigs
```

Comment:
Agreed. Will touch that up if this pull request goes forward; otherwise will open a touchup PR. Thanks!